app/src/main/res/values/integers.xml

```diff
@@ -8,5 +8,5 @@
     <!-- EditorInfo.IME_ACTION_* is defined by the framework. -->
     <integer name="ime_action_search">3</integer>
 
-    <integer name="webview_initial_zoom">100</integer>
+    <integer name="webview_initial_zoom">300</integer>
```

Comment:
No action needed: unfortunately, this is the unintuitive part of the resources system. :( Our least primary device gets the base file.